### ISSUE TITLE
[REQUEST] Data saver mode/ compressed images

### ISSUE THREAD

**Issue by Benwor28** [Paperback-iOS/app#188]
**Description**
Mangadex website has a data saver mode that compresses the images. It makes things load faster at the cost of image quality. I’m hoping for a way to have that setting implemented into the app.
**How would it improve the App?**
It would make images load faster when enabled which is nice
**Additional context**
![8A2A2389-74AA-4AFC-80B2-CF6CB805D10E](https://user-images.githubusercontent.com/64277468/90351085-c5b9ca80-e004-11ea-9d5e-1c4a6b3a71a3.jpeg)

**Comment by ConradWeiser** [comment]
Currently, if you are logged in to Mangadex inside of the app, and for your account (On their website) data-saver mode is enabled, that mode will be used by Paperback automatically while using the Mangadex source.

If you're suggesting a toggle in the app to switch it on or off, that will have to wait until sources are made stateful, which will likely happen at some point closer towards our implementation of trackers.

I'll leave this request open, since a in-app toggle of that mode is probably a good enhancement for Mangadex!
I hope the settings form submission on the website is simply a POST request that we can form a body for or something. We'll find out later when we get to it.

**Comment by newaccforhc** [comment]
No.. I have data saver mode enabled on website but, I'm still getting original images in paperback. 

I checked with a manga which has images 2000+ px.
MangaDex data saver mode changes such images to 1400px but, paperback is still sending the original images (difference between 2000px and 1400px is easily noticeable.

Here, I checked with this manga: [https://mangadex.org/chapter/1141130](https://mangadex.org/chapter/1141130)
